app/views/users/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<h2><%= @user.username %></h2>
+
+<div>
+  <%= @user.questions.count %> Question<%= @user.questions.count == 1 ? '' : 's'%>,
```

Comment:
Rails has a built in helper for this <%= pluralize @user.questions.count, 'question' %>